lib/clerk/resources.rb

```diff
@@ -6,6 +6,4 @@
 require_relative "resources/organizations"
 require_relative "resources/phone_numbers"
 require_relative "resources/sessions"
-require_relative "resources/sms_messages"
-require_relative "resources/users"
```

Comment:
❓ Was the `users` import unnecessary here?

Reply:
No no, fat fingered the delete! Fixed